js/entities/field.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Created by Sych on 04.04.2016.
+ */
+"use strict";
+
+class Field extends GameComponent {
+  constructor(options) {
+    super(options);
+    this._DOM = RenderHandler.renderField(this);
+
+    this._globalDeck = DeckHandler.initGlobalDeck();
+
+    this._gameRows = this._initGameRows();
+    this._bases = this._initBases();
+
+    this._deckHolder = new Deck({
+      id: 'deck',
+      selector: 'deck',
+      parentDOM: this._DOM,
+      deck: DeckHandler.getCardsForDeck(24, this._globalDeck)
+    });
+
+    this._openedDeckHolder = new Deck({
+      id: 'opened-deck',
+      selector: 'opened-deck',
+      parentDOM: this._DOM,
+      deck: DeckHandler.getCardsForDeck(0, this._globalDeck)
+    });
+
+    this._deckHolder.getDOM().addEventListener('deckClicked', this.onDeckClicked.bind(this));
+    this._addEventListenersForCardHolders('cardDnD', this.onCardDnD);
+
+  }
+
+  getDeckHolder() {
+    return this._deckHolder;
+  }
+
+  getOpenedDeckHolder() {
+    return this._openedDeckHolder;
+  }
+
+  getGameRows() {
+    return this._gameRows;
+  }
+
+  getGameRowByNumber(i) {
+    return this._gameRows[i];
+  }
+
+  getBases() {
+    return this._bases;
+  }
+
+  getBaseByNumber(i) {
+    return this._bases[i];
+  }
```

Comment:
You can use next construction: get baseByNumber(i) {...} and after that you can call this field like "nameOfObject.baseByNumber(i)" I think it's more sorter and besides it's normal way to declare get methods.